contracts/orbital-core/src/account_types.rs

```diff
@@ -0,0 +1,56 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{ensure, DepsMut, StdError, StdResult, Uint64};
+
+use crate::state::OrbitalDomainConfig;
+
+#[cw_serde]
+pub enum AccountConfigType {
+    Polytone {
+        note: String,
+        timeout: Uint64,
+    },
+    ICA {
+        connection_id: String,
+        channel_id: String,
+        timeout: Uint64,
+    },
+}
+
+impl AccountConfigType {
+    pub fn try_into_domain_config(self, deps: &DepsMut) -> StdResult<OrbitalDomainConfig> {
```

Comment:
nit: can use Deps here instead of DepsMut

Reply:
changed it to `api` directly to be more explicit